app/src/test/java/nerd/tuxmobil/fahrplan/congress/schedule/LayoutCalculatorTest.kt

```diff
@@ -203,11 +203,12 @@ class LayoutCalculatorTest {
 
         val session1 = createSession(date = conferenceDate, startTime = startTime1, duration = duration1)
         val session2 = createSession(date = conferenceDate, startTime = startTime2, duration = duration2)
-        val sessions = listOf(session1, session2)
+        val sessions = mutableListOf(session1, session2)
```

Comment:
There's no need for a list now. Use `session1` and `session2` directly.

Reply:
:heavy_check_mark: Refactored.